wsdl-core/src/main/java/io/ballerina/xsd/core/visitor/VisitorUtils.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
```

Comment:
```suggestion
 * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com)
```
Let's check and update other places